src/simulation/Simulation.cpp

```diff
@@ -487,6 +505,10 @@ Simulation::haveAllExternalized(uint32 num, uint32 maxSpread)
     uint32_t min = UINT32_MAX, max = 0;
     for (auto it = mNodes.begin(); it != mNodes.end(); ++it)
     {
+        if (mPausedNodes.find(it->first) != mPausedNodes.end())
+        {
+            continue;
+        }
```

Comment:
looks like `Simulation::crankNode` doesn't handle paused nodes at the moment. But also, I'm not really sure we need the pause functionality at all. We can just restart the node, allowing the network to close a few ledgers in between.

Reply:
And by "stop the node", I mean using `Simulation::removeNode` (it's used in a few other tests like this already)

Reply:
Resolving this: the test was updated to spin up a new node.